openshift_pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -0,0 +1,58 @@
+[id='creating-applications-with-cicd-pipelines']
+= Creating applications with CI/CD Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/openshift-pipelines-attributes.adoc[]
+:context: creating-applications-with-cicd-pipelines
+toc::[]
+
+With {op-title}, you can create a customized CI/CD solution to build, test and deploy your application.
+
+To create a fully-fledged, self-serving CI/CD Pipeline for an application, you have to perform the following tasks:
+
+* Create custom Tasks, or install existing reusable Tasks.
+* Create a Pipeline and Pipeline Resources to define the delivery Pipeline of your application.
+* Create a PipelineRun to instantiate and invoke the Pipeline.
+* Add Triggers to capture any events in the source repository.
+
+This section uses the `pipelines-tutorial` example to demonstrate the above tasks. The example uses a simple application which consists of:
+
+* A front-end interface `vote-ui`, with the source code in link:https://github.com/openshift-pipelines/vote-ui[`ui-repo`] Git repository.
+* A back-end interface `vote-api`, with the source code in link:https://github.com/openshift-pipelines/vote-api[`api-repo`]  Git repository.
+
+
+[discrete]
+== Prerequisites
+
+* You have access to {product-title} cluster.
+
+* You have installed link:TBD[{op-title}] using the {op-title} Operator listed in the OpenShift OperatorHub. Once installed, it is applicable to the entire cluster.
+
+* You have installed the link:TBD[Tekton CLI].
```

Comment:
1. when will in update these TBD?
2. are we going to call it "Tekton CLI" even though Tekton is the name of upstream project?

Reply:
1. I will add the xrefs for now and comment them out. Since the modules for installing pipelines and tkn cli are not yet merged, the links wont work for now.
2. It now renamed to OpenShift Pipelines (tkn) CLI

Reply:
These lines will be un-commented once we have merged the dependent modules. We will create another PR for the same.